### ISSUE TITLE
Canvas containing 3D model too large

### ISSUE THREAD

**Issue by lovelord83** [r-o-b-o-t-o/azerothcore-armory#11]
### Describe your feature request or suggestion in detail

Canvas frame containing 3D model is too large (unusefull)

### Describe a possible solution to your suggestion

reduce canvas width

**Comment by angrytoilet** [comment]
Agreed - if there is a simple place we can edit this, i would be more then happy to edit and do it myself.
